test/helpers/wait-for-update.ts

```diff
@@ -21,15 +21,15 @@ window.waitForUpdate = initialCb => {
     if (queue.length) {
       let hasError = false
       try {
-        job.wait ? job(shift) : job()
+        job.wait ? job(shift) : (job ? job() : null)
```

Comment:
let's not make any javascript changes for now, if the object can be `undefined` let's add an `@ts-expect-error` and we can do a mass processing later on